grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -978,7 +978,7 @@ class OrderController {
 
     def createCombinedShipment = {
         def orderInstance = Order.get(params.orderId)
-        if (!orderInstance.orderItems.find {it.quantityRemainingToShip != 0 }) {
+        if (!orderInstance.orderItems.find {it.quantityRemainingToShip != 0 && it.orderItemStatusCode != OrderItemStatusCode.CANCELED }) {
```

Comment:
@awalkowiak @DominikaHatala Here's another case where it would really good to have the logic in a service or domain method so we can easily add unit tests when the time comes. Please start putting yourself in that frame of mind when you're writing code. 

FWIW this logic could also happen in the createFromOrder() method. If you find that you're about to create an order with no items, throw an exception.